master/buildbot/util/git.py

```diff
@@ -50,7 +50,26 @@ def getSshCommand(keyPath, knownHostsPath):
 
 class GitMixin:
 
-    def setupGit(self):
+    def setupGit(self, logname=None):
+        if logname is None:
+            logname = 'GitMixin'
+
+        if self.sshHostKey is not None and self.sshPrivateKey is None:
+            config.error('{}: sshPrivateKey must be provided in order use sshHostKey'.format(
+                logname))
+            self.sshHostKey = None
```

Comment:
this shows as uncovered, because config.error raises by default. Can you please fix the coverage?

Reply:
Fixed now.